### PR TITLE
Add support to OpenBSD

### PR DESCRIPTION
**Pull request by gonzalo-**
@kesselb  I hope this time everything is good :D

### CODE REVIEW

**Comment by kesselb** [comment]
Thanks @gonzalo- :+1: 

Linter and DCO looks good.

**Comment by gonzalo-** [comment]
No love for this? :D

File: admin_manual/installation/example_openbsd.rst [hunk 1]
```diff
@@ -0,0 +1,236 @@
+.. _openbsd_installation_label:
+
+Nextcloud does not have official OpenBSD or other BSDs support
```
Comment (kesselb) [inline admin_manual/installation/example_openbsd.rst:3]: image

> Nextcloud does not have official OpenBSD or other BSDs support

Is now a navigation item. Can you use the box again?

**Review by kesselb — CHANGES_REQUESTED** [review]

File: admin_manual/installation/example_openbsd.rst (hunk 1 above)
Comment (kesselb) [reply admin_manual/installation/example_openbsd.rst:3]:
```
.. _openbsd_installation_label:

Example installation on OpenBSD
===============================

.. warning::

    Nextcloud does not have official OpenBSD or other BSDs support

In this install tutorial we will be deploying Nextcloud on a minimal OpenBSD with our own httpd(8), PHP, PostgreSQL and redis (for -stable or -current are the same steps).
```

image

**Review by kesselb — APPROVED** [review]

**Comment by kesselb** [comment]
@gonzalo- mind to fix the DCO?

**Comment by gonzalo-** [comment]
Uff I did it from github website can I fix it from here? it is really important?

**Comment by kesselb** [comment]
`git commit --amend --no-edit --signoff`
`git push --force-with-lease origin master`

**Comment by gonzalo-** [comment]
Done!

**Comment by kesselb** [comment]
Thanks @gonzalo- :+1:

**Pull request merged**